store/extensions/Robosort/extension.json

```
@@ -0,0 +1,52 @@
+{
+  "title": "Robosort",
+  "description": "Sort your wired stacks (automatically).",
+  "authors": [
+    {
+      "name": "Roboroads",
+      "hotel": ".nl",
+      "username": "Roboroads",
+      "discord": "Roboroads",
```

```suggestion
      "discord": "roboroads",
```
Discord usernames are lowercase